### ISSUE TITLE
REST API for closure compiler throwing a CORS error (again)

### ISSUE THREAD

**Issue by ScottHuston** [google/closure-compiler#3869]
Access to XMLHttpRequest at 'https://closure-compiler.appspot.com/compile' from origin 'https://dev.XXXXXX.com' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource.

Resumption of https://github.com/google/closure-compiler/issues/3850#issue-965294693 which @brad4d fixed after the release last month.

**Comment by ScottHuston** [comment]
When visiting https://closure-compiler.appspot.com/ or https://closure-compiler.appspot.com/compile from a browser I'm getting a 503

Error Over Quota

This application is temporarily over its serving quota.  Please try again later.

**Comment by ScottHuston** [comment]
After a temporary reprieve of working on Saturday, we're back to CORS error reported in the browser with the REST call and 503 over quota reported from the website.

**Comment by ScottHuston** [comment]
Do we have some hope of a resolution on this? @rishipal @brad4d

**Comment by ScottHuston** [comment]
It seems that https://closure-compiler.appspot.com/ works in the AM and by afternoon is over quota.

Since these are both Google projects can we get the quota limit raised?  Is that the issue?

**Comment by worldoptimizer** [comment]
Yes, I am also using the API … low frequency, but it is a dependency at this point to minify code for my repositories on GitHub. Any chance this could be solved with a little help from Google? Much appreciated.

**Comment by worldoptimizer** [comment]
It seems like the API is coming back every so often. I don't know what people are doing with it, but I suspect something shady. It's kind of sad to see individual users (or bots run by said users) doing this and ruining this great and free service from Google.

**Comment by ScottHuston** [comment]
> It seems like the API is coming back every so often. I don't know what people are doing with it, but I suspect something shady. It's kind of sad to see individual users (or bots run by these users) doing this and ruining this great and free service from Google for the rest of us. It needs to be reliable to be part of a workflow.

If we could get a response from someone on the project, that would be a start.  Feels like no one can be bothered.

Come on guys (@rishipal @brad4d @lauraharker )  Can you at least do some expectation management?

**Comment by lauraharker** [comment]
@concavelenz was working on getting the billing quota increased.

(We have an internal ticket tracking this since there are some non-public links)
